packages/react-native-windows-init/src/Cli.ts

```diff
@@ -415,14 +415,14 @@ function getRNWInitVersion(): string {
   return '';
 }
 
-function setExit(exitCode: ExitCode, error?: String): void {
+function setExit(exitCode: number, error?: string): void {
   if (!process.exitCode || process.exitCode === ExitCode.SUCCESS) {
     Telemetry.client?.trackEvent({
       name: 'init-exit',
       properties: {
         durationInSecs: process.uptime(),
         msftInternal: isMSFTInternal(),
-        exitCode: ExitCode[exitCode],
+        exitCode: exitCode,
```

Comment:
we used to log the string, this changes behavior to log the number - can we please restore this one?